specification/resources/resource-manager/Microsoft.Authorization/stable/2025-01-01/examples/createOrUpdatePolicyDefinition.json

```diff
@@ -0,0 +1,95 @@
+{
+  "parameters": {
+    "subscriptionId": "ae640e6b-ba3e-4256-9d62-2993eecfa6f2",
+    "policyDefinitionName": "ResourceNaming",
+    "api-version": "2024-05-01",
```

Comment:
>2024-05-01 [](http://example.com/codeflow?start=20&length=10)

update for all examples

Reply:
It is already 2025-01-01. You are looking at an outdated version of the files